src/mui/list/List.js

```diff
@@ -17,6 +17,10 @@ import translate from '../../i18n/translate';
 
 const filterFormName = 'filterForm';
 
+const styles = {
+    noResults: { padding: 20, display: 'flex', alignItems: 'flex-end' },
```

Comment:
I dont think that display: 'flex', alignItems: 'flex-end' is needed - the padding 20 I thought kept it roughly inline with the table header it was replacing